hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerBalancerStartSubcommand.java

```diff
@@ -74,13 +74,54 @@ public class ContainerBalancerStartSubcommand extends ScmSubcommand {
           "(for example, '26' for 26GB).")
   private Optional<Long> maxSizeLeavingSourceInGB;
 
+  @Option(names = {"-b", "--balancing-iteration-interval"},
```

Comment:
How about we call this `--balancing-iteration-interval-minutes` to indicate we're expecting time in minutes? We can also add that to the description to make it super clear. Same for the other time related options.